src/lib/error.ts

```diff
@@ -1,6 +1,8 @@
 export const error = (e: unknown) => {
   if (e instanceof TypeError) {
     console.error(e.message)
+  } else if (typeof e === 'string') {
```

Comment:
I think that instead of checking if the error is a string here, we should just do something like

```
} else {
  console.error(e.toString())
}
```

That way, it will catch other errors as well. Does anyone know if there was a reason for suppressing error outputs and limiting it to only TypeError?